tests/test_integration.py

```diff
@@ -480,3 +480,41 @@ def test_downgrade_api(self):
         ok_(response)
         eq_(response.status, 200)
         ok_(isinstance(api._encoder, JSONEncoder))
+
+@skipUnless(
+    os.environ.get('TEST_DATADOG_INTEGRATION', False),
+    'You should have a running trace agent and set TEST_DATADOG_INTEGRATION=1 env variable'
+)
+class TestRateByService(TestCase):
+    """
+    Check we get feedback from the agent and we're able to process it.
+    """
+    def setUp(self):
+        """
+        Create a tracer without workers, while spying the ``send()`` method
+        """
+        # create a new API object to test the transport using synchronous calls
+        self.tracer = get_dummy_tracer()
+        self.api_json = API('localhost', 8126, encoder=JSONEncoder())
+        self.api_msgpack = API('localhost', 8126, encoder=MsgpackEncoder())
+
+    def test_send_single_trace(self):
+        # register a single trace with a span and send them to the trace agent
+        self.tracer.trace('client.testing').finish()
+        trace = self.tracer.writer.pop()
+        traces = [trace]
+
+        # [TODO:christian] when CI has an agent that is able to process the v0.4
```

Comment:
ok will keep track of that

Reply:
https://github.com/DataDog/dd-trace-py/pull/359/commits/1812ba0c8966f227a47afe0dd73ef217fa19b42c